README.md

```diff
@@ -121,6 +123,103 @@ This will output a pack directory in the `/tmp` directory.
 > By default, the forge does quality checks on the pack that may require docker.
 > You can disable these via `-n`
 
+Via API pack generation can be done via:
+
+``` Python
+from grand_challenge_forge.forge import generate_challenge_pack
+from Pathlib import Path
+
+qc = []
+generate_challenge_pack(
+    context={"challenge": {...}}
+    output_path=Path("dist/"),
+    quality_control_registry=qc,
+    force=False,
```

Comment:
What does `Force` do?

Reply:
If the output directory (dynamically named) already exists, setting `force=True` will remove it. It makes sense in the CLI interface where `force` is the common nomenclature for this. For a python API I think `allow_existing` is better suited.

Reply:
Renamed it `delete_existing`, kept `force` in the CLI.